examples/tfengine/full.yaml

```diff
@@ -87,6 +87,20 @@ templates:
   output_ref: "folder-prod.dir"
   data:
     DISPLAY_NAME: "team1"
+- name: "project-example-prod-networks"
+  recipe_path: "{{$BASE}}/folder/project.yaml"
+    output_ref: "folder-prod-team1.dir"
+  data:
+    PROJECT_ID: "example-prod-networks"
+    RESOURCES:
+      COMPUTE_NETWORKS:
```

Comment:
Why not have a separate "networks" recipe?

Reply:
IMO being able to define a single project and its resources through one recipe is cleaner. Else you would have way too many templates here and it would be very messy.